app/config/routes.js

```diff
@@ -0,0 +1 @@
+// routes?
```

Comment:
@a-ashbeck is it possible we need some routes? I'm just grasping at straws. I will start looking into the API tomorrow